proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,567 @@
+2. It takes place at least once a year, with invitations including a
+   preliminary agenda is sent to all core and extraordinary members at
+   least six weeks prior. The members may submit additional agenda items to
+   the board until two weeks before the assembly. The Board will publish the
+   final agenda online.
+3. An extraordinary General Assembly may be called if requested by at least
+   20 % of core members.
+4. The General Assembly has a quorum if the invitation was submitted in proper
+   form. Should the number of pro and contra votes should be equal, an
+   application is considered as dismissed.
+5. A 3/4 majority of votes is required to amend the Articles of Association or
+   decide the liquidation of the Association.
+6. Core members unable to attend may be represented by another member with
+   written authorization.
+7. Participation and voting in the General Assembly must also be possible via
+   electronic communication. By resolution of the Board, a General Assembly
+   may be held exclusively by these means.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is
+   responsible for all duties unless otherwise assigned.
+2. It elects the Board from a number of core members and can deselect board
+   members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges
+   the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of
+   the Association.
+6. Namely the annual accounts and the annual report have to be presented in
+   written format to the General Assembly for decision making and discharge
+   of the board. It calls on an auditor, who belong neither to the board nor
+   any other panel appointed by the board and who is not employed by the
+   association, in order to audit the accounting and annual report and
+   relate their results to the General Assembly. The auditors have access
+   to all of the Association’s accounting documents.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer
+   elected for a 3-year term.
+2. The Board’s duties and meeting procedures are stipulated in the by-laws.
+   The Chairman is responsible for following the purpose of the association,
+   complying with the Articles of Association and delegation of tasks. The
+   second chairperson represents the Chairman in his absence and verifies the
+   resolutions of the Board and the General Assembly. The treasurer manages
+   the finances of the association, keeps the cash books and the list of
+   members.
+3. The Board decides on all matters unless a General Assembly decision is
+   required. It implements the decisions taken by the General Assembly.
+4. Each Board member has the right to represent the Association.
```

Comment:
The board” it is recommended to increase that each board member may represent individually.